### PR TITLE
OSDOCS-8517 Node bug text for 4.15 GA RNs

### PR DESCRIPTION
**Pull request by cbippley**
Version(s): 4.15

Issue: [OSDOCS-8517](https://issues.redhat.com/browse/OSDOCS-8517)

Link to docs preview: [Scroll to Node section under Bug Fixes](https://71916--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-bug-fixes)

QE review:
Not needed

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -1502,6 +1502,18 @@ With this release, instead of returning an overflow `servfail` error, the CoreDN
...
+
+* Previously, the kubelet was running with the `unconfined_service_t` SELinux type. As a consequence, all our plugins failed to deploy due to an Selinux denial. With this fix, the kubelet now runs with the `kubelet_exec_t` SELinux type. As a result, plugins deploy as expected. (link:https://issues.redhat.com/browse/OCPBUGS-20022[*OCPBUGS-20022*])
+
+* Previously, the kubelet was running with the `unconfined_service_t` SELinux type. As a consequence, all our plugins failed to deploy due to an Selinux denial. With this fix, the kubelet now runs with the `kubelet_exec_t` SELinux type. As a result, plugins deploy as expected. (link:https://issues.redhat.com/browse/OCPBUGS-20022[*OCPBUGS-20022*])
+
+* Remove the automatic image removal on an upgrade. Now, when Openshift performs a minor upgrade, the container images won't be automatically removed. This caused issues in pre-pulling images. The images will be instead subject to kubelet's image garbage collection, which will trigger based on disk usage. (link:https://issues.redhat.com/browse/OCPBUGS-25228[*OCPBUGS-25228*])
```
Comment (tmalove) [inline release_notes/ocp-4-15-release-notes.adoc:1515]:
```suggestion
* Remove the automatic image removal on an upgrade. Now, when Openshift Container Platform performs a minor upgrade, the container images will not be automatically removed. This caused issues in pre-pulling images. The images will be instead subject to kubelet's image garbage collection, which will trigger based on disk usage. (link:https://issues.redhat.com/browse/OCPBUGS-25228[*OCPBUGS-25228*])
```
See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#openshift-glossary-of-terms

**Review by tmalove — APPROVED** [review]

File: release_notes/ocp-4-15-release-notes.adoc [hunk 2]
```diff
@@ -1516,6 +1516,18 @@ With this release, instead of returning an overflow `servfail` error, the CoreDN
...
+
+* Previously, because of a default setting (`container_use_dri_devices, true`), containers were unable to use dri devices. With this fix, containers can use dri devices as expected. (link:https://issues.redhat.com/browse/OCPBUGS-24042[*OCPBUGS-24042*])
+
+* Previously, the kubelet was running with the `unconfined_service_t` SELinux type. As a consequence, all our plugins failed to deploy due to an Selinux denial. With this fix, the kubelet now runs with the `kubelet_exec_t` SELinux type. As a result, plugins deploy as expected. (link:https://issues.redhat.com/browse/OCPBUGS-20022[*OCPBUGS-20022*])
+
+* Previously, the kubelet was running with the `unconfined_service_t` SELinux type. As a consequence, all our plugins failed to deploy due to an Selinux denial. With this fix, the kubelet now runs with the `kubelet_exec_t` SELinux type. As a result, plugins deploy as expected. (link:https://issues.redhat.com/browse/OCPBUGS-20022[*OCPBUGS-20022*])
```
Comment (jldohmann) [inline release_notes/ocp-4-15-release-notes.adoc:1527]: This looks to be a duplicated entry for the one just above it

**Review by jldohmann — COMMENTED** [review]
Some minor changes needed before merge

File: release_notes/ocp-4-15-release-notes.adoc (hunk 2 above)
Comment (cbippley) [reply release_notes/ocp-4-15-release-notes.adoc:1527]: Whoops, thanks!

**Pull request merged**